content/en/blog/2021-01-12_gender_bias_and_mobility.md

```diff
@@ -19,29 +19,29 @@ author:
+Data is not objective truth - despite what we data analysts would like to believe. Analyses reflect the perspective from which they were created. If a team is too uniform in their experiences, then the diverse perspectives of different age groups, genders, or ethnicities are easily forgotten. Such bias is often observed in relation to gender, where most analyses focus on the 'default male'[^1], leading to serious, even deadly consequences for women[^2]. For example, crash dummies tend to represent the average male size, which is [one of the reasons why women are more likely to be injured or killed in a crash](https://www.consumerreports.org/car-safety/crash-test-bias-how-male-focused-testing-puts-female-drivers-at-risk/). Women are also more likely to be [misdiagnosed when having a heart attack](https://www.leeds.ac.uk/news/article/4269/women_more_at_risk_of_dying_after_a_heart_attack) because studies have mostly been done with male participants, who show very different symptoms to women. There is a growing awareness of this gender bias in data science, with great books written on this issue by Caroline Criado Pérez[^1], Catherine D'Ignazio and Lauren F. Klein[^3], and Cathy O'Neil[^4].
 
-Having no (or not enough) data about women isn't the only issue: If there is data on men and women alike, it's often not gender disaggregated, which is necessary to reveal gender related differences. For example, gender disaggregated data is necessary to understand gender biases in working hours (including both paid and unpaid work): Gender disaggregated data does not exist for all countries, but in almost all countries where it exists, there is a clear trend that women work longer - on average 50 min more each day (see The [Global Gender Gap Report 2016](http://www3.weforum.org/docs/GGGR16/WEF_Global_Gender_Gap_Report_2016.pdf)). Another example where gender disaggregated data is important is the area of mobility behavior, which we want to investigate closer in this blog post.
+Having no (or not enough) data about women isn't the only issue: If there is data on men and women alike, it's often not gender disaggregated, which is necessary to reveal gender related differences. For example, gender disaggregated data is necessary to understand gender biases in working hours (including both paid and unpaid work). In almost all countries where gender disaggregated work data exists there is a clear trend that women work longer - on average 50 min more each day (see The [Global Gender Gap Report 2016](http://www3.weforum.org/docs/GGGR16/WEF_Global_Gender_Gap_Report_2016.pdf)). Another example where gender disaggregated data is important is the area of mobility behavior, which we want to investigate closer in this blog post.
```

Comment:
I put the sentence in this way, so the intro to that paragraph makes sense: 
'Having no (or not enough) data about women isn't the only issue: If there is data on men and women alike, it's often not gender disaggregated'
therefore I'd leave it the way it is. What do you think?

Reply:
Yep, sounds good to me.